### DISCUSSION TITLE
🧾List of Currently Supported Games

### DISCUSSION THREAD

**Discussion by paranormalraccoon** [etournity/etournity#29] [Game Support]
1. Brawlhalla (1v1, Single Elimination)
2. [Your game](https://github.com/etournity/etournity/discussions/new?category=game-support)?

**Comment by DavidAyliffe** [comment]
Please add support for:
Minecraft Capture the Flag (5 vs 5)
Call of Duty (5 vs 5)
Chess (1 vs 1)
